src/MapBox.js

```diff
@@ -1,23 +1,276 @@
+          description: 'San Francisco, California'
+      }],
+      
+      line:[{
+      paint:{
+          lineColor : '#888',
+          lineWidth: 8
+      },
+      coordinates: [
+          [-122.48369693756104, 37.83381888486939],
+          [-122.48348236083984, 37.83317489144141],
+          [-122.48339653015138, 37.83270036637107],
+          [-122.48356819152832, 37.832056363179625],
+          [-122.48404026031496, 37.83114119107971],
+          [-122.48404026031496, 37.83049717427869],
+          [-122.48348236083984, 37.829920943955045],
+          [-122.48356819152832, 37.82954808664175],
+          [-122.48507022857666, 37.82944639795659],
+          [-122.48610019683838, 37.82880236636284],
+          [-122.48695850372314, 37.82931081282506],
+          [-122.48700141906738, 37.83080223556934],
+          [-122.48751640319824, 37.83168351665737],
+          [-122.48803138732912, 37.832158048267786],
+          [-122.48888969421387, 37.83297152392784],
+          [-122.48987674713133, 37.83263257682617],
+          [-122.49043464660643, 37.832937629287755],
+          [-122.49125003814696, 37.832429207817725],
+          [-122.49163627624512, 37.832564787218985],
+          [-122.49223709106445, 37.83337825839438],
+          [-122.49378204345702, 37.83368330777276]
+      ],
+      title: 'line#1',
+      description: 'line#1 description here'
+      
+      },{
+          paint:{
+              lineColor : '#222',
+              lineWidth: 8
+          },
+          coordinates: [
+              [-112.48369693756104, 37.83381888486939],
+              [-112.48348236083984, 37.83317489144141],
+              [-112.48339653015138, 37.83270036637107],
+              [-112.48356819152832, 37.832056363179625],
+              [-112.48404026031496, 37.83114119107971],
+              [-112.48404026031496, 37.83049717427869],
+              [-112.48348236083984, 37.829920943955045],
+              [-112.48356819152832, 37.82954808664175],
+              [-112.48507022857666, 37.82944639795659],
+              [-112.48610019683838, 37.82880236636284],
+              [-112.48695850372314, 37.82931081282506],
+              [-112.48700141906738, 37.83080223556934],
+              [-112.48751640319824, 37.83168351665737],
+              [-112.48803138732912, 37.832158048267786],
+              [-112.48888969421387, 37.83297152392784],
+              [-112.48987674713133, 37.83263257682617],
+              [-112.49043464660643, 37.832937629287755],
+              [-112.49125003814696, 37.832429207817725],
+              [-112.49163627624512, 37.832564787218985],
+              [-112.49223709106445, 37.83337825839438],
+              [-112.49378204345702, 37.83368330777276]
+          ],
+          title: 'line#1',
+          description: 'line#1 description here'
+      }],
+  }
+}
+
+ ******************************************************************************************************************************************/
```

Comment:
It is good to move this sample usage to README.md later.
Because this will not be delivered to the end user.

Reply:
Let me move this sample codes to README.md